src/CppUTestExt/IEEE754ExceptionsPlugin.cpp

```diff
@@ -71,7 +71,7 @@ void IEEE754ExceptionsPlugin::enableInexact()
 
 bool IEEE754ExceptionsPlugin::checkIeee754ExeptionFlag(int flag)
 {
-    return fetestexcept(flag);
+    return !! fetestexcept(flag);
```

Comment:
Apparently an interesting, different fix to the bool problem ^^. This pull doesn't seem to be based on the latest master though, or this change would have been unnecessary. Shouldn't there be a merge conflict?